Index: app/frontend/views/visitenkartenModal.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><div class=\"visitenkartenModal\">\r\n    <h3>Visitenkarten Modal</h3>\r\n    <?php\r\n    if($visitenkarte->id_user != $_SESSION['user']['id_user']) {\r\n        echo '<button type=\"button\" class=\"btn btn-primary\" data-toggle=\"modal\" data-target=\"#visitenkartenModal\">\r\n            User melden\r\n        </button>';\r\n    }\r\n    echo \"Name: \" . $visitenkarte->firstname . \" \" . $visitenkarte->lastname . \"<br>\";\r\n    echo \"Beschreibung: \" . $visitenkarte->description . \"<br>\";\r\n    echo \"Interessen: <br>\";\r\n    echo \"<ul>\";\r\n    foreach ($visitenkarte->interests as $interest) {\r\n        echo \"<li>\" . $interestModel->getInterestName($interest) . \"</li>\";\r\n    }\r\n    echo \"</ul>\";\r\n    echo \"Kontaktinformationen: <br>\" . $visitenkarte->contactInformation;\r\n\r\n    ?>\r\n</div>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/frontend/views/visitenkartenModal.php b/app/frontend/views/visitenkartenModal.php
--- a/app/frontend/views/visitenkartenModal.php	(revision 4f94412d401fbb03d4d3e080463ec6fc3d808d93)
+++ b/app/frontend/views/visitenkartenModal.php	(date 1657618926752)
@@ -1,5 +1,4 @@
 <div class="visitenkartenModal">
-    <h3>Visitenkarten Modal</h3>
     <?php
     if($visitenkarte->id_user != $_SESSION['user']['id_user']) {
         echo '<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#visitenkartenModal">
Index: app/frontend/views/profile.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><h1>\r\n    Profile\r\n</h1>\r\n\r\n<?php\r\n// print own visitenkarte\r\ninclude 'app/frontend/views/visitenkartenModal.php';\r\n?>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/frontend/views/profile.php b/app/frontend/views/profile.php
--- a/app/frontend/views/profile.php	(revision 4f94412d401fbb03d4d3e080463ec6fc3d808d93)
+++ b/app/frontend/views/profile.php	(date 1657618944840)
@@ -2,7 +2,60 @@
     Profile
 </h1>
 
+<div>
+<a href="?t=frontend&request=profilepicedit" class="btn">Profilbild bearbeiten</a><br>
+</div>
+
+<!-- Button trigger modal -->
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalCenter">
+    Visitenkarte
+</button>
+
+<!-- Modal -->
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLongTitle">Visitenkarte</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <?php
+                include 'visitenkartenModal.php'
+                ?>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Schließen</button>
+            </div>
+        </div>
+    </div>
+</div>
+<br>
+
+
 <?php
-// print own visitenkarte
-include 'app/frontend/views/visitenkartenModal.php';
-?>
\ No newline at end of file
+//Show Profile Information 
+if($profile->id_user != $_SESSION['user']['id_user']) {
+    echo '<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#ProfileTestModal">
+        User melden
+    </button>';
+}
+echo "Name: " . $profile->firstname . " " . $profile->lastname . "<br>";
+echo "Geschlecht: " . $profile->gender . "<br>";
+echo "scoreHost: " . $profile->scoreHost . "<br>";
+echo "scoreAttendee: " . $profile->scoreAttendee . "<br>";
+echo "Beschreibung: " . $profile->description . "<br>";
+echo "Interessen: <br>";
+echo "<ul>";
+foreach ($profile->interests as $interest) {
+    echo "<li>" . $interestModel->getInterestName($interest) . "</li>";
+}
+echo "</ul>";
+echo "Kontaktinformationen: <br>" . $profile->contactInformation;
+
+?>
+<div>
+<a href="?t=frontend&request=profileedit" class="btn">Profil bearbeiten</a><br>
+</div>
\ No newline at end of file
